### ISSUE TITLE
ETTm1的结果异常问题

### ISSUE THREAD

**Issue by Soramishy** [plumprc/MTS-Mixers#2]
![image](https://user-images.githubusercontent.com/127716923/232984418-54f081b5-a2a0-4059-aba8-21f332221b2c.png)
你好，我按照程序训练默认的ETTM1数据，选用MTSMixers模型，最后输出的结果pdf特别离谱，然后我改用SCINet，发现结果好了很多，但是依然比较离谱，请问我是哪一步做错了么，为什么会出现这样的结果
![image](https://user-images.githubusercontent.com/127716923/232985173-ffcf08d1-3ae2-44da-ba12-93f1768b0119.png)

**Comment by plumprc** [comment]
您好，其实这些预测结果就应该很离谱，因为现存的做长时间序列预测的工作本质上都是在学习周期性，当测试集出现之前没有过的周期模式时，他理论上就不太可能做到准确预测。可以关注我们的另一个仓库 [https://github.com/plumprc/RTSF](https://github.com/plumprc/RTSF)，稍后会在这里分享我们关于长时间序列预测的观点

**Comment by rdyan0053** [comment]
有测ETTh1数据集的吗？这是我复现论文的结果：
test 2785
mse:0.3126, mae:0.4615, R2:-1.7379

这是我的参数设置：
Namespace(is_training=1, model='MTSMixer', data='ETTh1', root_path='./dataset/', data_path='ETTh1.csv', features='MS', target='OT', freq='h', checkpoints='./checkpoints/', seq_len=336, label_len=48, pred_len=96, individual=False, seg=20, rev=False, norm=True, fac_T=False, sampling=2, fac_C=False, refine=False, mat=0, embed_type=0, enc_in=7, dec_in=7, c_out=7, d_model=512, n_heads=1, e_layers=2, d_layers=1, d_ff=2048, moving_avg=25, factor=1, dropout=0.05, embed='timeF', activation='gelu', output_attention=False, do_predict=False, num_workers=10, itr=1, train_epochs=10, batch_size=16, patience=3, learning_rate=0.001, loss='mse', lradj='type1', use_amp=False, use_gpu=True, gpu=0, use_multi_gpu=False, devices='0,1,2,3', test_flop=False)

下图分别对于结果中的0.pdf，80.pdf，160.pdf
<img width="400" alt="image" src="https://github.com/plumprc/MTS-Mixers/assets/53009617/15e5b661-d4ea-4548-939b-7dd570fbb8ac">

<img width="395" alt="image" src="https://github.com/plumprc/MTS-Mixers/assets/53009617/682e42fc-4b51-4248-ba29-5dc43d86347b">

<img width="407" alt="image" src="https://github.com/plumprc/MTS-Mixers/assets/53009617/ee33f14a-4256-47a3-a0f2-ac813cf07d32">

感觉好像效果没有DLinear好，是不是我哪个参数设置的不对

**Comment by plumprc** [comment]
> 这是我的参数设置： Namespace(is_training=1, model='MTSMixer', data='ETTh1', root_path='./dataset/', data_path='ETTh1.csv', features='MS', target='OT', freq='h', checkpoints='./checkpoints/', seq_len=336, label_len=48, pred_len=96, individual=False, seg=20, rev=False, norm=True, fac_T=False, sampling=2, fac_C=False, refine=False, mat=0, embed_type=0, enc_in=7, dec_in=7, c_out=7, d_model=512, n_heads=1, e_layers=2, d_layers=1, d_ff=2048, moving_avg=25, factor=1, dropout=0.05, embed='timeF', activation='gelu', output_attention=False, do_predict=False, num_workers=10, itr=1, train_epochs=10, batch_size=16, patience=3, learning_rate=0.001, loss='mse', lradj='type1', use_amp=False, use_gpu=True, gpu=0, use_multi_gpu=False, devices='0,1,2,3', test_flop=False)
> 感觉好像效果没有DLinear好，是不是我哪个参数设置的不对

您好，MTS-Mixers 的实验设置是 96-x，DLinear 原始论文中的设置为 336-x，关于序列长度对预测结果的影响请参考 [DLinear](https://arxiv.org/abs/2205.13504) 附录实验以及 [RTSF](https://arxiv.org/abs/2305.10721) 的说明
